solidity/contracts/vault/TBTCVault.sol

```diff
@@ -207,8 +207,9 @@ contract TBTCVault is IVault, Ownable {
     /// @notice Allows the governance to finalize vault upgrade process. The
     ///         upgrade process needs to be first initiated with a call to
     ///         `initiateUpgrade` and the `UPGRADE_GOVERNANCE_DELAY` needs to
-    ///         pass. Once the upgrade is finalized, the new vault will become
-    ///         an owner of TBTC token.
+    ///         pass. Once the upgrade is finalized, the new vault becomes the
+    ///         owner of TBTC token and receives a whole Bank balance of this
```

Comment:
owner of TBTC token and receives a whole Bank balance of this
-> owner of the TBTC token and receives the whole Bank balance of this

Reply:
See 91fbcc52bac31cf3ec9dc2058c8e10c1bdc20d6a please.